src/astUtils/visitors.spec.ts

```diff
@@ -119,6 +120,7 @@ describe('astUtils visitors', () => {
                 'Block:2',                // while block
                 'PrintStatement:3',       // print 6
                 'ForStatement:1',         // for a = 1 to 10
+                'AssignmentStatement:2',  // a = 1
```

Comment:
@elsassph My walk function emits this statement. I think this is correct, but your test didn't include it.